ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```diff
@@ -49,6 +49,9 @@ images:
       as:
       - registry.access.redhat.com/ubi8/ubi:latest
   to: e2e-test
+promotion:
+  name: "4.14"
+  namespace: ocp
```

Comment:
I don't think we need this. Cincinnati (not the operator) is already built after each commit and pushed to `quay.io/app-sre/cincinnati`

Reply:
Actually, she should promote the `cincinnati` operand image here, such as
```console
- dockerfile_path: dist/Dockerfile.deploy/Dockerfile
  inputs:
    ubi:
      as:
      - registry.access.redhat.com/ubi8/ubi:latest
  to: cincinnati
```
And then, refer to this operand image in the operator, like
```console
  driver-operand:
    name: "4.14"
    namespace: ocp
    tag: cincinnati
```

Reply:
Of course, we'd better to use the `ci` namespace instead.

Reply:
> Cincinnati (not the operator) is already built after each commit and pushed to quay.io/app-sre/cincinnati

Is the image built before each o./cincinnati PR merge or post merge?

Reply:
`quay.io/app-sre/cincinnati` is built after merge - by a different pipelline (app-interface), not ci-operator/openshift ci. Annoying thing I have discovered yesterday is that it gets built and pushed but there's no floating `latest` tag which we need. I'll see if we can configure app-interface to create that tag as well.

Of course all images are also built before merge by ci-operator/openshift ci. My point is that to enable pre-merge testing we don't need `promotion`.

Reply:
Sometimes we might enable some new features in cincinnati but delay cincinnati-operator changes as maybe we don't have bandwidth. In this case, to pre-merge test cincinnati changes, we need to use the upstream way https://github.com/openshift/cincinnati/blob/master/docs/user/running-cincinnati.md which requires cincinnati image. So imo cincinnati promotion is good to help us pre-merge test cincinnati. Please correct me if I'm wrong:)

Reply:
Promotion means "build images **after** merge and persist them": same as app-interface (except that one little detail of missing `latest` tag). 

To pre-merge test cincinnati changes, you have a pending cincinnati PR and need to build cincinnati image from the PR code. That already happens and is not related to promotion.

Reply:
Also, it seems that we have some cincinnati image persisted in here (no idea how it gets there without promotion)

```
  cincinnati_operand_latest:
    name: cincinnati-build-root
    namespace: cincinnati-ci
    tag: deploy
```

Reply:
Thanks for the explanation!

I just promoted the `cincinnati-ci/cincinnati-build-root:deploy` as it's referenced by the bundle image https://github.com/openshift/release/pull/42053/files#diff-3e4c9ff59bb36d000ea248cbef83680065c76329d216d38ce61c00fb798d3e94R33. Not sure if it makes sense:)

Reply:
`cincinnati-ci/cincinnati-build-root:deploy` is promoted here: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cincinnati/openshift-cincinnati-master__customrust.yaml#L48-L50

Reply:
Right, there's a promoting variant...